packages/client/examples/browser/package.json

```diff
@@ -9,6 +10,6 @@
     "vite": "^2.3.7"
   },
   "dependencies": {
-    "web3.storage": "^1.0.0"
+    "web3.storage": "file:../.."
```

Comment:
When published can it please depend on a real version so users don't have to npm install in the root as well as here?

Reply:
yes, this was just for testing until the latest module release, which now!